schemas/cs3/tbl/status.ksy

```diff
@@ -0,0 +1,211 @@
+meta:
+  id: status
+  endian: le
+  encoding: UTF-8
+doc: |
+  Monster status.
+  Includes stat bases and growths; elemental, status, and unbalance
+  efficacies; sepith and item drops; and more.
+
+  Extremely similar to the player character status table (status_p).
+
+  Present in t_mons.tbl
+seq:
+  - id: script
+    type: strz
+  - id: texture
+    type: strz
+  - id: model
+    type: strz
+
+  - id: unknown_float_1
```

Comment:
Some of these are related to size, such as model size and hitbox size, but I don't remember which is which